pkg/provider/azure_utils.go

```diff
@@ -447,27 +447,23 @@ func getResourceByIPFamily(resource string, isDualStack, isIPv6 bool) string {
 }
 
 // isFIPIPv6 checks if the frontend IP configuration is of IPv6.
-func (az *Cloud) isFIPIPv6(service *v1.Service, fip *network.FrontendIPConfiguration, isInternal bool) (bool, error) {
-	if isInternal {
```

Comment:
Could you please add some comments here? because I would never know that the privateipaddress is empty when the lb is not internal lb until I checked the fip configs of existing cluster.

Reply:
Updated.